index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="starting-page"> 
```

Comment:
[OPTIONAL] As suggested by previous reviewer [here](https://github.com/Shri-sakthi-v/mobile-first-portfolio/pull/1#discussion_r1108538837), I think it would be better to cover the whole height of the page. One way could be adding `height:100vh` to  `starting-page` class 🥇 I know you can do that 💪

Reply:
Thanks a lot kanza. I became so exhausted by the changes.
Really I am so happy for approving my pull request.
Regards shri sakthi